cmd/makedocs/check-docs_test.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"go/parser"
+	"go/token"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+	"testing"
+)
+
+var docCheckers []checkerDoc
+
+func parseFiles() []checkerDoc {
```

Comment:
I though you would use same function as `main.go`.
Maybe `parseFiles` can accept path argument?
Please, move it to main.go and use it in both places.

Reply:
> I though you would use same function as `main.go.`
What function?

Reply:
One I though you would write to avoid code duplication between test and main.go.
If we're testing main.go, it should use same function as the main.go